pullsbury/handlers/slack.py

```diff
@@ -11,6 +11,7 @@ def __init__(self, event, config):
         self.emojis = config.get('SLACK_EMOJIS')
         self.slack_icon = config.get('SLACK_ICON')
         self.repo_blacklist = config.get('REPO_BLACKLIST')
+        self.custom_emoji_mapping = config.get('CUSTOM_EMOJI_MAPPING', {})
```

Comment:
Can you add this to the `settings.py` file?  Also, it should be slacked prefixed:
`SLACK_CUSTOM_EMOJI_MAPPING`